.github/workflows/black.yml

```
@@ -0,0 +1,71 @@
+name: black
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  format:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+  
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install black
+        run: pip install black
+
+      - name: Black
+        run: black manim tests example_scenes
+
+      - name: Check for modified files
+        shell: bash
+        id: git-check
+        run: echo ::set-output name=modified::$(if git diff-index --quiet HEAD --; then echo "false"; else echo "true"; fi)
+
+      - name: Print Black Version if Failed
+        shell: bash
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          blackVer=$(black --version)
+          blackVersion=$(python -c "print('${blackVer}'.split()[2])")
+          date=$(python -c "import time;print(time.localtime().tm_mday)")
+          pushBranch="black-${blackVersion}-${date}"
+          git config user.name "github-actions"
+          git config user.email "actions@github.com"
+          git checkout -B "$pushBranch"
+          git add .
+          git commit -m "Running Black ON Master"
```

Nitpick (like, really a nitpick)
```suggestion
          git commit -m "Running Black on Master"
```

---

I did for fun and was intentional. lol

---

I will change it soon.